examples/policies/l3/requires/endpoints.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "l3-rule"
+specs:
+  - description: "For endpoints with env=prod, allow if source also has label role=frontend"
+    endpointSelector:
+      matchLabels:
+        env: prod
+    ingress:
+    - fromEndpoints:
+      - matchLabels:
+          role: frontend
```

Comment:
Maybe this is still intended as a draft but it doesn't seem to have the requires rules at all :)

Reply:
That's intentional, to highlight the difference between `toRequires`+`toEndpoints` and `toEndpoints`+`toEndpoints`. I've updated the original post with a preview of what the rendered version looks like.